components/patient-data/ui/src/main/resources/PhenoTips/UIX_Field__charts.xml

```diff
@@ -116,9 +116,13 @@
 #set ($targetClassName = 'PhenoTips.MeasurementsClass')
 #if ($xcontext.action != 'export' &amp;&amp; ($xcontext.action == 'edit' || $doc.getObject($targetClassName)))
   (% class="growth-charts-section" %)(((
-    === $services.localization.render('phenotips.UIXField.growthCharts.heading') ===
-
-    #__expandTools()
+    #if($xcontext.action != 'view')
+      (% class="expandable" %)
```

Comment:
I see why you put the heading in two places, you have optional content both before and after it. I guess it's fine to put the heading in both branches of the `#if` then.